test/core/bin.js

```diff
@@ -132,7 +132,12 @@ _.each(fs.readdirSync(fixtureLoc), function (binName) {
       };
 
       var optionsLoc = testLoc + "/options.json"
-      if (fs.existsSync(optionsLoc)) _.merge(opts, require(optionsLoc));
```

Comment:
Test harness seemed fine? (passed both locally and travis) Can you clarify so I understand the arch better?

Regardless, I'll take a shot at moving it to an API test, unless told otherwise.